src/libData/AccountData/AccountStoreSC.tpp

```diff
@@ -580,7 +602,14 @@ bool AccountStoreSC<MAP>::ParseCreateContractOutput(
     outStr = {std::istreambuf_iterator<char>(in),
               std::istreambuf_iterator<char>()};
   }
-  LOG_GENERAL(INFO, "Output: " << std::endl << outStr);
+
+  LOG_GENERAL(
```

Comment:
You can avoid the check against max and just call `substr` in both cases, because `substr` will still work if you have less than max (http://www.cplusplus.com/reference/string/string/substr/)

Reply:
I used this check so that ellipses can be appended if the output string indeed has to be truncated.

Reply:
Ok